### ISSUE TITLE
rational needs some special handling as well

### ISSUE THREAD

**Issue by SamSaffron** [Shopify/bootsnap#12]
I fixed my boot errors by adding 

```
def require(path)
   return if path == 'thread.rb'.freeze || path == 'rational'
```

to the kernel_require monkey patch

looks like prev fix did not full fix the thread.rb issue, and rational special handling may be missing

**Comment by SamSaffron** [comment]
FYI here are some numbers from my quite old macbook pro for Discourse

https://gist.github.com/SamSaffron/d4f733108fe261815678b52b1a22f4b7 

(on my desktop its about 2x faster, no access to it here)

**Comment by burke** [comment]
Fixed in 0.2.9 (but I've pushed 0.2.10 because I figured out the other issue immediately after I pushed that version...
